src/Actions/Account/ListAccounts.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Choredo\Actions\Account;
+
+use Choredo\Actions\Behaviors;
+use Choredo\Entities\Account;
+use Choredo\EntityManagerAware;
+use Choredo\HasEntityManager;
+use Choredo\Output;
+use Choredo\Transformers\AccountTransformer;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Zend\Diactoros\Response\JsonResponse;
+use const Choredo\REQUEST_FAMILY;
+use const Choredo\REQUEST_FILTER;
+use const Choredo\REQUEST_PAGINATION;
+use const Choredo\REQUEST_SORT;
+
+class ListAccounts implements
+    EntityManagerAware,
+    Output\FractalAwareInterface,
```

Comment:
We should probably consider dropping `Interface` from our `Aware` things, so it's consistent.

Reply:
I agree, I thin it's better to do in a single pr though to get it all at once. I can do it in the cleanup branch in working on